### PR TITLE
Bumping Chart versions as pacman k8s version is much newer than our c…

### PR DESCRIPTION
**Pull request by Moumoutaru**
…harts

- Bumps chart version as pacman k8s version is much newer than our charts

### PR Description
Please replace this with a high level description of what this PR is supposed to accomplish.

### Checklist

 - [ ] Have you reviewed and updated the chart default values if necessary?
 - [ ] Have you reviewed and updated the chart documentation if necessary?
 - [ ] Does your branch follow the naming convention of `{chartNameWithDashes}-v{versionString}-{optionalPatchVersion}`?
 - [ ] Have you bumped the version in the chart's `Chart.yaml`?

### Tagged Releases
Please remember to make a tagged release after merging your PR that:

 - Has a tag name that matches your PR branch name (see above)
 - Has a description that summarizes the changes made

This makes it possible to use previous versions of the charts maintained here as new releases are published. Please see the release history of this repository for examples.

### CODE REVIEW

**Review by iblackman — DISMISSED** [review]

**Comment by bcdonadio** [comment]
Are we the only users of this Operator and CRDs? If we share them with Guardians, have we verified that we won't break any of their toys?

**Comment by iblackman** [comment]
> Are we the only users of this Operator and CRDs? If we share them with Guardians, have we verified that we won't break any of their toys?

If I'm not mistaken we use it on:
* mail-production
* harbor

And we have tags for that exact reason, to not break other people work

**Comment by jackiehpowerhrg** [comment]
Going to be watching this PR, Igor and I were talking, I might need this so I can get Netbox automatic backups working.

File: charts/crunchydata-pgo-db/templates/init-sql-cm.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+{{- if and .Values.databaseInitSQL .Values.databaseInitSQL.content }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ .Values.databaseInitSQL.name }}"
+data:
+  {{ .Values.databaseInitSQL.key }}: |-
+    {{ .Values.databaseInitSQL.content | nindent 4 }}
+{{- end }}
```
Comment (iblackman) [inline charts/crunchydata-pgo-db/templates/init-sql-cm.yaml:10]: Does this work when you do NOT have `databaseInitSQL.content` defined?

Comment (iblackman) [reply charts/crunchydata-pgo-db/templates/init-sql-cm.yaml:10]: I tested here and it seems to work well

**Review by iblackman — APPROVED** [review]

**Pull request merged**